xpsi/Instrument.py

```diff
@@ -353,7 +355,7 @@ def trim_response(self,
             cumsum[i] /= cumsum[i,-1]
 
         # Extract with the adapted percentile
-        under_tolerance = [ _np.all( cumsum[:,i] <= 1 - tolerance ) for i in range( cumsum.shape[1] ) ]
+        under_tolerance = [ _np.any( cumsum[:,i] <= 1 - tolerance ) for i in range( cumsum.shape[1] ) ]
```

Comment:
Are you sure this change (from your last commit) keeps the original logic (mentioned in the Changelog information) that the 
> cumulative of all the channels to reach the (1-tolerance) quantile
?

Isn't this code now requiring that at least one channel (but not necesserily more than that) is below the tolerance?

Reply:
If you decided to change the logic, I think the docstring and changelog information should be still updated.